Check if large support data files were installed

schaubh opened the pull request:
* **Tickets addressed:** bsk-623
* **Review:** By commit  <!-- Choose from: "by commit", "by file" -->
* **Merge strategy:** Merge (no squash)  <!-- Choose from: "merge (no squash)", "squash and merge" -->

## Description
If the user didn't install `lfs` before cloning the BSK repo, then the large spice data files are not installed
and some scripts will not run.  New users are finding it challenging to debug this situation.

The large spice file is now not included in the repo, but rather downloaded when running the `cmake` command.  This reduces the repo size and makes the Basilisk installation simpler.

## Verification
Tested running `conanfile.py` with and without having a large `de430.bsp` file and got the expected results.
If the hyperlink is no longer working an error message is thrown.
Did a clean build using `pip install .` , with the JPL file `de430.bsp` missing, and it installed correctly.

## Documentation
Documentation built without issues.  Updated release notes.

## Future work
None

patkenneally commented:
I think we should proceed in a different direction here. That is, we should remove the files from git-lfs and use CMake's download functionality to grab the latest version of 430.bsp from NAIF. I have tested this out and it works well. The git-lfs data bandwidth used by all user clones, CI clones, and CI clones on forks, counts towards this repos git-lfs bandwidth total. As the number of processes cloning the repo grows (inevitably) the amount of git-lfs bandwidth will also grow. We can avoid that cost by simply removing the files from git-lfs and downloading otherwise.

schaubh commented:
If you want to make a branch that implements this for me to test, I'm glad to update the install documentation, test this out, etc.  I agree that this would avoid some of the GitHub bandwidth issues we have run into with the other PR doing pip wheels, etc.

schaubh commented:
Updated the PR to now have `cmake` pull the JPL file.  Also removed two large un-used Spice files.  If the link is broken then an error is displayed to the console.

Mark2000 reviewed (approved):
Tested by deleting support files and rebuilding BSK. The bsp file was downloaded automatically. Good method for preventing a common new user error.
